client.go

```
@@ -29,14 +28,325 @@ func main() {
+}
+
+func createMappingForCapitals(sqlService sql.Service) {
+	// See: https://docs.hazelcast.com/hazelcast/5.1/sql/mapping-to-maps
+	fmt.Println("Creating a mapping...")
+	mappingQuery := fmt.Sprintf(`
+        CREATE OR REPLACE MAPPING capitals TYPE IMap
+		OPTIONS (
+			'keyFormat' = 'varchar',
+			'valueFormat' = 'varchar'
+		)
+	`)
+	ignored, err := sqlService.Execute(context.Background(), mappingQuery)
+	if err != nil {
+		panic(err)
+	}
+	defer ignored.Close()
+	fmt.Println("The mapping has been created successfully.")
+	fmt.Println("--------------------")
+}
+
+func clearCapitals(sqlService sql.Service) {
+	fmt.Println("Deleting data via SQL...")
+	ignored, err := sqlService.Execute(context.Background(), "DELETE FROM capitals")
+	if err != nil {
+		panic(err)
+	}
+	defer ignored.Close()
+	fmt.Println("The data has been deleted successfully.")
+	fmt.Println("--------------------")
+}
+
+func populateCapitals(sqlService sql.Service) {
+	fmt.Println("Inserting data via SQL...")
+	insertQuery := fmt.Sprintf(`
+        INSERT INTO capitals VALUES
+			('Australia','Canberra'),
+            ('Croatia','Zagreb'),
+			('Czech Republic','Prague'),
+			('England','London'),
+			('Turkey','Ankara'),
+			('United States','Washington, DC');
+	`)
+	ignored, err := sqlService.Execute(context.Background(), insertQuery)
+	if err != nil {
+		panic(err)
+	}
+	defer ignored.Close()
+	fmt.Println("The data has been inserted successfully.")
+	fmt.Println("--------------------")
+}
+
+func selectAllCapitals(sqlService sql.Service) {
+	fmt.Println("Retrieving all the data via SQL...")
+	ctx := context.Background()
+	result, err := sqlService.Execute(ctx, "SELECT * FROM capitals")
+	if err != nil {
+		fmt.Errorf("Querying: %w", err)
+	}
+	iterator, err := result.Iterator()
+	if err != nil {
+		fmt.Errorf("Acquaring iterator: %w", err)
+	}
+	for iterator.HasNext() {
+		row, err := iterator.Next()
+		if err != nil {
+			panic(err)
+		}
+		country, err := row.Get(0)
```

Let's not ignore errors, unless there's an important reason for that. In that case we should add a comment about why the error was ignored. Same comment for similar code.

---

Related `Get()` and `GetByColumnName()` errors are handled.